lib/grp.gi

```diff
@@ -4401,6 +4401,12 @@ InstallMethod( GroupWithGenerators,
 function( gens )
 local G,typ;
 
+  if IsDomain(gens) then
+    Info( InfoPerformance, 1,
+    "Calling `GroupWithGenerators' on a domain is very inefficient.\n",
+    "#I  Use the list of generators of the domain instead.");
```

Comment:
This hardcodes an assumption about how the Info message is printed. It's better to use two `Info` statements instead, or else put everything in one line.

Reply:
Sure. Updated to resolve both comments.